ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -88,9 +88,11 @@ TFCore::getModelPath ()
  * @brief	load the tf model
  * @note	the model will be loaded
  * @return 0 if OK. non-zero if error.
- *        -1 if the pb file is not loaded.
- *        -2 if the input properties are different with model.
- *        -3 if the Tensorflow session is not created.
+ *        -1 if the modelfile is not valid(or not exist).
```

Comment:
BTW, is the negative value is better? :) Which better between the negative values and positive values?
From my experience, most of the developers keep the existing 'man' style. 
Fore example, ubuntu16.04$ man sched_setaffinity, Then please check "RETURN VALUE section.

Reply:
There is no reason marking it negative.
In my opinion, defining and using the proper error codes looks better to understand and analyze log.